packages/embed/README.md

```diff
@@ -71,31 +71,32 @@ Or from admin panel URL:
+| hideHeaders                  | boolean     | hide header that appears when you have a question group, or a long question (does not apply to Chat UI)                    | `false`                                                       |
+| opacity                      | number      | form background opacity, number from 0 (fully transparent) 100 (fully opaque)                                              | `100`                                                         |
+| disableAutoFocus             | boolean     | disable form auto focus when loaded                                                                                        | `false`                                                       |
+| open                         | string      | open embed based on user action (see below)                                                                                | `undefined`                                                   |
+| openValue                    | number      | based on `open` (see below)                                                                                                | `undefined`                                                   |
+| enableSandbox                | boolean     | enable [sandbox mode](https://help.typeform.com/hc/en-us/articles/360029295952) (disables submissions and tracking)        | `false`                                                       |
+| tooltip                      | string      | display tooltip text next to the button (popover only)                                                                     | `undefined`                                                   |
+| notificationDays             | number      | display red notification dot, hide for given number of days since popover is open (popover only)                           | `undefined`                                                   |
+| onReady                      | function    | fires when the form is loaded                                                                                              | `undefined`                                                   |
+| onSubmit                     | function    | fires when user submits the form                                                                                           | `undefined`                                                   |
+| onQuestionChanged            | function    | fires when user navigates between form questions                                                                           | `undefined`                                                   |
+| shareGoogleAnalyticsInstance | function    | Allows to share the Google instance of the page with the embedded typeform                                                 | `false`                                                       |
```

Comment:
Please update the description to match:
```
| shareGoogleAnalyticsInstance | function    | shares Google Analytics instance of the host page with embedded typeform                                                 | `false`                                                       |
```

PS: I am intentionally not suggesting this as Github "suggestion" here. Please update it in your IDE and run prettier on the file :)